dev/puppetserver.conf.sample

```diff
@@ -5,7 +5,7 @@ global: {
 product: {
     update-server-url: "http://localhost/"
     name: {group-id: puppetlabs.dev
-           artifact-id: puppet-server}
+           artifact-id: puppetserver}
```

Comment:
The artifact-id that we've been using to report to dujour has been "puppetserver" rather than "puppet-server" for some time now.  Can't remember when it changed but it was several releases back.  I think the value we use in the dev sample file isn't too relevant anyway.  We use that in conjunction with `puppetlabs.dev` rather than `puppetlabs` (production) to hopefully filter out the `dev` related check-ins somewhat, AFAIR.